static/src/javascripts-legacy/projects/common/modules/atoms/story-questions.js

```diff
@@ -35,6 +39,29 @@ define([
                     bean.on(el, 'click', askQuestion);
```

Comment:
If you want we can change it so that clicks are delegated to the surrounding `js-view-tracking-component`, instead of having an event listener for each question.